TodoApp/src/TodoApp.Database/ItemsRepository.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using TodoApp.Contracts;
+using TodoApp.Contracts.Models;
+
+namespace TodoApp.Database
+{
+    internal class ItemsRepository : IItemRepository
```

Comment:
Replace functions starting with return statement with lambda bodied expressions (similarly to `ItemsController`) [this applies to the whole solution of course, however, it was most significant here]